backend/import/toBoosterCard.js

```diff
@@ -48,7 +47,12 @@ const toBoosterCard = (setCode) => (mtgjsonCard, index, rawCards) => {
     manaCost: manaCost || "",
     rarity: upperFirst(rarity),
     url: url || `https://api.scryfall.com/cards/${identifiers.scryfallId}?format=image`,
-    identifiers: identifiersMini,
+    identifiers: {
+      scryfallId: identifiers.scryfallId,
+      mtgoId: identifiers.mtgoId
+      // needed for exporting cards to MTGO .dek
+      // NOTE: not all cards seem to have this identifier..?
```

Comment:
@ZeldaZach I think this point is for you. How do MTGJson produce the identifier mtgoId ?

Reply:
I just deleted my data and restarted the server. Send to build and grab the mtgoId fine

Reply:
Oh someone told me that some cats eg unset were never released on mtgo so they have no id there!